src/test/java/teammates/common/datatransfer/questions/FeedbackRankQuestionDetailsTest.java

```diff
@@ -0,0 +1,58 @@
+package teammates.common.datatransfer.questions;
+
+import org.testng.annotations.Test;
+
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackRankQuestionDetails}.
+ */
+public class FeedbackRankQuestionDetailsTest extends BaseTestCase {
+    @Test
+    public void testValidateMinOptionsToBeRanked_sameValidValues_shouldReturnTrue() {
+        int testValue = 100;
+        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
+        feedbackRankQuestionDetails.setMinOptionsToBeRanked(testValue);
+        assertEquals(feedbackRankQuestionDetails.minOptionsToBeRanked, testValue);
+    }
+
+    @Test
+    public void testValidateMaxOptionsToBeRanked_sameValidValues_shouldReturnTrue() {
+        int testValue = 100;
+        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
+        feedbackRankQuestionDetails.setMaxOptionsToBeRanked(testValue);
+        assertEquals(feedbackRankQuestionDetails.maxOptionsToBeRanked, testValue);
+    }
+
+    @Test
+    public void testValidateMaxOptionsToBeRanked_differentValidValues_shouldReturnTrue() {
+        int testValue = 100;
+        int testDiffValue = 101;
+        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
+        feedbackRankQuestionDetails.setMaxOptionsToBeRanked(testValue);
+        assertNotEquals(feedbackRankQuestionDetails.maxOptionsToBeRanked, testDiffValue);
+    }
+
+    @Test
+    public void testValidateDuplicatesAllowed_validValues_shouldReturnTrue() {
+        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
+        feedbackRankQuestionDetails.setAreDuplicatesAllowed(true);
+        assertEquals(feedbackRankQuestionDetails.areDuplicatesAllowed, true);
+
+        feedbackRankQuestionDetails.setAreDuplicatesAllowed(false);
+        assertEquals(feedbackRankQuestionDetails.areDuplicatesAllowed, false);
+
+        feedbackRankQuestionDetails.setAreDuplicatesAllowed(true);
+        assertNotEquals(feedbackRankQuestionDetails.areDuplicatesAllowed, false);
```

Comment:
These checks aren't necessary, they make the code more complex (uncertainty in what you're testing) and are already covered by the above equal tests.